build.sbt

```diff
@@ -130,19 +128,49 @@ lazy val commonSettings = Seq(
 // General Settings
 inThisBuild(List(
   organization := "org.typelevel",
-  organizationName := "Typelevel Contributors",
-  developers := List(
-    Developer("rossabaker", "Ross A. Baker", "ross@rossabaker.com", url("https://github.com/rossabaker"))
-  ),
+  organizationName := "Typelevel",
+  publishGithubUser := "rossabaker",
+  publishFullName := "Ross A. Baker",
+  baseVersion := "0.3",
 
   homepage := Some(url("https://github.com/typelevel/case-insensitive")),
   startYear := Some(2020),
   licenses := Seq("Apache-2.0" -> url("https://www.apache.org/licenses/LICENSE-2.0.html")),
+  scmInfo := Some(ScmInfo(url("https://github.com/typelevel/case-insensitive"),
+    "git@github.com:typelevel/case-insensitive.git")),
 
-  pomIncludeRepository := { _ => false},
-  scalacOptions in (Compile, doc) ++= Seq(
-      "-groups",
-      "-sourcepath", (baseDirectory in LocalRootProject).value.getAbsolutePath,
-      "-doc-source-url", "https://github.com/typelevel/case-insensitive/blob/v" + version.value + "€{FILE_PATH}.scala"
+  githubWorkflowTargetTags ++= Seq("v*"),
+  githubWorkflowPublishTargetBranches := Seq(RefPredicate.StartsWith(Ref.Tag("v"))),
+  githubWorkflowBuildPreamble := Seq(
+    WorkflowStep.Use("olafurpg", "setup-gpg", "v3"),
```

Comment:
Is this still necessary?

Reply:
I sincerely don't know :-(

Reply:
Well, there's one way to find out. *yoink*

Reply:
It might also be that it passes now but fails at publish time.